### DISCUSSION TITLE
Cold weather kit high temperature

### DISCUSSION THREAD

**Discussion by zachliu** [jgyates/genmon#1214] [Questions]
Anyone got the cold weather kit from Amazon? The [Generac ones](https://a.co/d/iUs9ryJ) are over $200. The knockoffs are $100 and less. I got the latter and it's the [Amazon's choice](https://a.co/d/7eCH8CR). However, it heats up everything to a whopping 200F and maintains the temperature. It's definitely bad right? I wonder what temperature do the real Generac kits produce 🤔 

I have a temperature sensor taped on the oil filter heat pad

![image](https://github.com/user-attachments/assets/685f4cf9-c6c5-4d49-83d8-b4bf5dbd886b)

![image](https://github.com/user-attachments/assets/e2be1768-bce0-4a7e-a1a8-0bf8a05f6645)

Ambient temperature

![image](https://github.com/user-attachments/assets/76badb7b-d091-40bc-b162-f20bce9920ba)

**Comment by Jkh62** [comment]
My local installer did not recommend one on my install , NE Ohio.
Just cycled my gen under load for 30 minutes every 1st of the month , its low 20's F , I use Mobil 1 synthetic oil and no issues for cold weather cranking or start up. YMMV.

**Comment by jtw1979** [comment]
Not sure about these vs genuine Generac ones but they do get very hot as they are 240V. 200F sounds about right as they have to counter the freezing cold to keep your oil from thickening up. Just curious if this oil temp functionality is only on Evolutions? My Nexus 8kW has the cold weather kit but don't get readings for it.
